compiler-rt/test/sanitizer_common/TestCases/fopen64_nullptr.c

```diff
@@ -0,0 +1,8 @@
+// Check that fopen64(NULL, "r") is ok.
+
```

Comment:
One of your earlier changes removed the 'RUN' line, so this test will have an error such as
```
******************** TEST 'SanitizerCommon-lsan-x86_64-Linux :: fopen64_nullptr.c' FAILED ********************
Test has no 'RUN:' line
```